pyproject.toml

```diff
@@ -10,9 +10,9 @@ authors = ["kevinheavey <kevinheavey123@gmail.com>"]
 [tool.poetry.dependencies]
 python = "^3.9"
 construct-typing = "^0.5.1"
-solana = "^0.29.0"
+solana = ">=0.29.0,<30.0"
```

Comment:
I think `>=0.29.0,<30.0` is equivalent to `^0.29.0`

Reply:
Noted, thanks i was not sure, and wanted to be explicit. Happy to resolve this in another pr